examples/ShowPlayer.cpp

```diff
@@ -31,6 +31,7 @@
 #include <ola/base/SysExits.h>
 #include <ola/client/ClientWrapper.h>
 #include <ola/client/OlaClient.h>
+#include <climits>
```

Comment:
I think `#include <stdint.h>` should provide INT64_MAX too and we've already used that successfully on lots of platforms.

Reply:
I actually pulled in `<climits>` because we needed `UINT_MAX`, and that's not something that `<stdint.h>` is specified to provide.

Reply:
Though in hindsight it's only needed for clamping, so I should have guarded that with an `#if... #endif` too....

Reply:
Ah, we use `#include <limits.h>` for that (I think) in https://github.com/OpenLightingProject/ola/blob/ea3a18a925f123c8771ebc6c60327678e99ddcf2/plugins/pathport/PathportPlugin.cpp#L113